Source/WebKit/UIProcess/mac/WebPageProxyMac.mm

```diff
@@ -547,7 +547,7 @@ static inline bool expectsLegacyImplicitRubberBandControl()
 
     if (auto selectedMenuItem = [menuTarget selectedMenuItem]) {
         NSInteger tag = selectedMenuItem.tag;
-        if (contextMenu.openInPreviewIndex && *contextMenu.openInPreviewIndex == tag)
+        if (contextMenu.openInPreviewTag && *contextMenu.openInPreviewTag == tag)
```

Comment:
I know you're just rewording things, but [`std::optional<T>::operator==(const optional<T>& opt, const U& value)`](https://en.cppreference.com/w/cpp/utility/optional/operator_cmp) does the null check for free